client/src/components/Creator.js

```diff
@@ -1,8 +1,150 @@
 import React, { Component } from "react";
+import { Container, Row, Col, Button, Card, Table } from "react-bootstrap";
+import { ImCopy, ImTwitter, ImTelegram } from "react-icons/im";
+import { TwitterShareButton, TelegramShareButton } from "react-share";
+import "./Creator.css";
+
+function getBalance() {
+  return "0";
+}
 
 class Creator extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  getValue(weiVal) {
+    const val = (weiVal / Math.pow(10, 18)).toFixed(3);
+    return val + " ETH";
+  }
+
+  getRealTime(timestamp) {
+    var utcSeconds = parseInt(timestamp);
+    var d = new Date(0); // The 0 there is the key, which sets the date to the epoch
+    d.setUTCSeconds(utcSeconds);
+    return d.toLocaleString();
+  }
+
   render() {
-    return <div>Creator page</div>;
+    const balance = getBalance() + " aUSDC";
+    const fanLink =
+      "www.blossym.org/fan?creatorAddress=" + this.props.connectedWallet;
```

Comment:
We can omit `www` and specify `https://`: `https://blossym.org/fan?creatorAddress=`